package.json

```diff
@@ -54,6 +54,8 @@
     "@sentry/tracing": "^7.8.1",
     "@silvermine/videojs-airplay": "^1.1.0",
     "@silvermine/videojs-chromecast": "^1.3.3",
+    "@stripe/react-stripe-js": "^1.10.0",
+    "@stripe/stripe-js": "^1.35.0",
```

Comment:
The changes make sense visually;  few comments:

- :small_red_triangle:  Missing new `yarn.lock` commit.
- :small_red_triangle:  Might need Miko to re-test everything on `pk_live`.  I tested on `pk_test` again, but just like before, I didn't encounter the issues that he did.
- :small_blue_diamond: The name length error sometimes dissapears as you type. Even when it appears, clicking on the disabled red button will enable it.
    - <img width="150" alt="image" src="https://user-images.githubusercontent.com/64950861/186883235-4c27e16e-b3f6-4727-a074-fa01a9ca93d2.png">         <img width="150" alt="image" src="https://user-images.githubusercontent.com/64950861/186883327-e95aed1e-ed5d-43df-a7e1-bc946e96b87a.png">
- :small_blue_diamond: the card number input could be vertically centered, but this is super minor.
    - <img width="176" alt="image" src="https://user-images.githubusercontent.com/64950861/186882545-0616cd6e-542f-42d3-a655-1a5a38f73dec.png">

Reply:
For whatever reason, i still get stuck on loading when adding a card, which is why Rafael has been trying a few more things. Card gets added, but requires a refresh.

Reply:
@tzarebczan try again and if it still happens let me know if you get a correct `customer` `status` call with the card details since this should be the problem and not updating it, since you showed the `confirm` and `setupIntent` both go through